src/harness/reference_models/iap/iap_example.py

```diff
@@ -0,0 +1,132 @@
+# Test IAP calculation for GWPZ, PPA, FSS and ESC Sensor incumbent types.
+# Expected result is dictionary containing post IAP aggregate interference
+# value at a protection constraint. 
+# =============================================================================
+
+import sas_objects
+import json
+import os
+import iap
+import time
+from reference_models.interference import interference as interf
+
+cbsd_0 = json.load(open(os.path.join('test_data', 'cbsd_0.json')))
+cbsd_1 = json.load(open(os.path.join('test_data', 'cbsd_1.json')))
+cbsd_2 = json.load(open(os.path.join('test_data', 'cbsd_2.json')))
+cbsd_3 = json.load(open(os.path.join('test_data', 'cbsd_3.json')))
+cbsd_4 = json.load(open(os.path.join('test_data', 'cbsd_4.json')))
+cbsd_5 = json.load(open(os.path.join('test_data', 'cbsd_5.json')))
+cbsd_6 = json.load(open(os.path.join('test_data', 'cbsd_6.json')))
+cbsd_7 = json.load(open(os.path.join('test_data', 'cbsd_7.json')))
+cbsd_8 = json.load(open(os.path.join('test_data', 'cbsd_8.json')))
+cbsd_9 = json.load(open(os.path.join('test_data', 'cbsd_9.json')))
+cbsd_10 = json.load(open(os.path.join('test_data', 'cbsd_10.json')))
+cbsd_11 = json.load(open(os.path.join('test_data', 'cbsd_11.json')))
+cbsd_12 = json.load(open(os.path.join('test_data', 'cbsd_12.json')))
+cbsd_13 = json.load(open(os.path.join('test_data', 'cbsd_13.json')))
+cbsd_14 = json.load(open(os.path.join('test_data', 'cbsd_14.json')))
+cbsd_15 = json.load(open(os.path.join('test_data', 'cbsd_15.json')))
+cbsd_16 = json.load(open(os.path.join('test_data', 'cbsd_16.json')))
+cbsd_17 = json.load(open(os.path.join('test_data', 'cbsd_17.json')))
+cbsd_18 = json.load(open(os.path.join('test_data', 'cbsd_18.json')))
+cbsd_19 = json.load(open(os.path.join('test_data', 'cbsd_19.json')))
+cbsd_20 = json.load(open(os.path.join('test_data', 'cbsd_20.json')))
+cbsd_21 = json.load(open(os.path.join('test_data', 'cbsd_21.json')))
+cbsd_22 = json.load(open(os.path.join('test_data', 'cbsd_22.json')))
+cbsd_23 = json.load(open(os.path.join('test_data', 'cbsd_23.json')))
+
+
+fss_0 = json.load(open(os.path.join('test_data', 'fss_0.json')))
+fss_1 = json.load(open(os.path.join('test_data', 'fss_1.json')))
+
+ppa_0 = json.load(open(os.path.join('test_data', 'ppa_0.json')))
+ppa_1 = json.load(open(os.path.join('test_data', 'ppa_1.json')))
+ppa_2 = json.load(open(os.path.join('test_data', 'ppa_2.json')))
+ppa_3 = json.load(open(os.path.join('test_data', 'ppa_3.json')))
+
+esc_0 = json.load(open(os.path.join('test_data', 'esc_0.json')))
+
+gwpz_0 = json.load(open(os.path.join('test_data', 'gwpz_0.json')))
+gwpz_1 = json.load(open(os.path.join('test_data', 'gwpz_1.json')))
+
+pal_0 = json.load(open(os.path.join('test_data', 'pal_0.json')))
+pal_1 = json.load(open(os.path.join('test_data', 'pal_1.json')))
+pal_2 = json.load(open(os.path.join('test_data', 'pal_2.json')))
+pal_3 = json.load(open(os.path.join('test_data', 'pal_3.json')))
+
+sas_uut = sas_objects.FullActivityDump("", "", "", "")
```

Comment:
An empty FullActivityDump should be created by FullActivityDump({}). However as the FAD records are added right below It would be better to do:

```python
sas_uut = FullActivityDump({'cbsd': [cbsd_0, cbsd_1, ... etc ]})
```

Likewise for the other FADs.